app/Http/Controllers/HomeController.php

```diff
@@ -113,6 +121,7 @@ public function showIndex()
             'previousDate'         => $startDate->copy()->subDays($daysToShow)->toDateString(),
             'nextDate'             => $startDate->copy()->addDays($daysToShow)->toDateString(),
             'pageTitle'            => Setting::get('app_name').' Status',
+            'favicon'              => $favicon,
```

Comment:
Good idea. Do you think I should move just the default value or the full logic for determining which favicon to show? Full logic makes sense to me.